nano/nano_node/entry.cpp

```diff
@@ -1234,19 +1240,10 @@ int main (int argc, char * const * argv)
 			}
 
 			// Start new node
-			nano::node_config config2 (24001, logging);
-			// Config override
-			std::vector<std::string> config_overrides;
-			auto config (vm.find ("config"));
-			if (config != vm.end ())
-			{
-				config_overrides = config->second.as<std::vector<std::string>> ();
-			}
+			nano::node_config config2 = daemon_config.node;
+			config1.peering_port = 24001;
 			if (!config_overrides.empty ())
```

Comment:
This if condition seems not required anymore?

Reply:
I suppose not :)